atmoz-sftp.yaml

```diff
@@ -0,0 +1,82 @@
+#nolint:valid-pipeline-git-checkout-commit,valid-pipeline-git-checkout-tag
+package:
+  name: atmoz-sftp
+  version: 0.0.0_git20250226
+  epoch: 0
+  description: Various scripts from https://github.com/atmoz/sftp to help run SFTP in a container
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - bash
+      - openssh-client # Includes `sftp` command
+      - openssh-keygen # used in entrypoint script
+      - openssh-pam
+      - openssh-sftp-server
+      - posix-libc-utils # For `getent` command
+      - shadow
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libedit-dev
+      - linux-pam-dev
+      - openssh-client
+      - openssh-keygen
+      - openssl-dev
+      - posix-libc-utils
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/atmoz/sftp
+      branch: master
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}/etc/ssh
+      mkdir -p ${{targets.contextdir}}/usr/local/bin
+      mkdir -p ${{targets.contextdir}}/usr/sbin
+      mkdir -p ${{targets.contextdir}}/var/run/sshd
+
+      cp -av files/sshd_config ${{targets.contextdir}}/etc/ssh/sshd_config
+      cp -av files/create-sftp-user ${{targets.contextdir}}/usr/local/bin/
+      cp -av files/entrypoint ${{targets.contextdir}}/
+
+      # Ensure correct SSHD setup
+      ln -sf /usr/sbin/sshd.pam ${{targets.contextdir}}/usr/sbin/sshd
```

Comment:
Since we are in the process of doing usrmerge, moving forward no binaries should be installed in /usr/sbin /bin or /sbin. It would be better to change this to /usr/bin